libs/wingsdk/src/target-sim/util.ts

```diff
@@ -71,6 +71,34 @@ export function makeSimulatorJsClient(filename: string, resource: Resource) {
 })()`;
 }
 
+export function makeSimulatorJsClientV2(filename: string, resource: Resource) {
+  const type = basename(filename).split(".")[0];
+  const env = makeEnvVarName(type, resource);
+  return `(function() {
+  let handle = process.env.${env};
```

Comment:
can we move to a helper and not emit all this code?

Reply:
https://github.com/winglang/wing/issues/6446